x/axelarnet/module.go

```diff
@@ -305,19 +298,49 @@ func (am AppModule) OnTimeoutPacket(
 		return err
 	}
 
-	t, err := types.PacketToTransfer(packet)
-	if err != nil {
-		return err
-	}
-	// requeue transfer
-	err = am.keeper.EnqueueTransfer(ctx, t)
-	if err != nil {
-		return err
-	}
-	return nil
+	return retryTransfer(ctx, am.keeper, am.transfer, am.channel, packet)
 }
 
 // NegotiateAppVersion implements the IBCModule interface
 func (am AppModule) NegotiateAppVersion(ctx sdk.Context, order channeltypes.Order, connectionID string, portID string, counterparty channeltypes.Counterparty, proposedVersion string) (version string, err error) {
 	return am.transferModule.NegotiateAppVersion(ctx, order, connectionID, portID, counterparty, proposedVersion)
 }
+
+func setTransferFailed(ctx sdk.Context, k keeper.Keeper, portID, channelID string, seq uint64) error {
+	transferID, ok := k.GetSeqIDMapping(ctx, portID, channelID, seq)
+	if !ok {
+		return nil
+	}
+
+	k.DeleteSeqIDMapping(ctx, portID, channelID, seq)
+
+	return k.SetTransferFailed(ctx, transferID)
+
+}
+
+func setTransferCompleted(ctx sdk.Context, k keeper.Keeper, portID, channelID string, seq uint64) error {
+	transferID, ok := k.GetSeqIDMapping(ctx, portID, channelID, seq)
+	if !ok {
+		return nil
+	}
+
+	k.DeleteSeqIDMapping(ctx, portID, channelID, seq)
+
+	return k.SetTransferCompleted(ctx, transferID)
+}
+
+func retryTransfer(ctx sdk.Context, k keeper.Keeper, transferK types.IBCTransferKeeper, channelK types.ChannelKeeper, packet ibcexported.PacketI) error {
+	transferID, ok := k.GetSeqIDMapping(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
```

Comment:
Add a comment that we only want to retry IBC transfers made by the network and not all timed out user transfers.